css/css-inline/text-box-trim/text-box-trim-nesting-001.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<title>Tests nested boxes are trimmed at text-over/text-under baselines</title>
+<meta description="Verify that a nested ex trimmed text has a height of 1ex" />
+<link rel="help" href="https://drafts.csswg.org/css-inline-3/#leading-trim">
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css" />
+<link rel="match" href="text-box-trim-nesting-001-ref.html">
+
+<style>
+div {
+  display: inline-flex;
+  font-size: 72px;
+  /*
+     for browser which don't support text-box-trim the text will not be at the same position
+     as the reference, but the height of the inline box should be the same
+  */
+  color: transparent;
+  border-top: 2px solid orange;
+}
+
+section {
+  /* limit theight to 1ex height using text-box-trim */
+  text-box-trim: both;
+  text-box-edge: ex alphabetic;
+}
+
+span {
+  border: 1px solid blue;
+  display: inline-block;
+  line-height: 3;
+  /* limit theight to 1ex height using text-box-trim */
+  text-box-trim: both;
+  text-box-edge: ex alphabetic;
+}
+</style>
+
+<div>
+  <section>
+    <span>e</span>
+  </section>
```

Comment:
Until https://github.com/w3c/csswg-drafts/issues/3956 is resolved I think it would be better to have two separate tests, one for SECTION and one for SPAN, rather than nesting them, because the 1px border is going to leak outside the 1ex reserved by SECTION, and it's not clear yet if we measure it or not when sizing the SECTION.

Reply:
Thanks @fantasai for your feedback

My motivation to place the SPAN inside a SECTION was to test the behaviour when nesting elements.

imho the current Safari TP implementation is not correct:
[![Safari TP](https://github.com/web-platform-tests/wpt/assets/4113649/f6caad0b-b4e7-4aae-9237-aa6349eea1ef)](https://bugs.webkit.org/show_bug.cgi?id=252161)

Would `outline` or `background` work better for this test?

I also created a separate test which tests a `span` without nesting here:
https://github.com/web-platform-tests/wpt/pull/40448